### PR TITLE
Authorization: added package tags for java, java readme

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: specification/authorization/resource-manager/readme.java.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+``` yaml $(tag) == 'package-2015-06-01-preview' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.authorization.v2015_06_01
+  output-folder: $(azure-libraries-for-java-folder)/authorization/resource-manager/v2015_06_01
+regenerate-manager: true
+generate-interface: true
+```
+
+
+### Tag: profile-hybrid-2019-03-01
```
Comment (lmazuel) [inline specification/authorization/resource-manager/readme.java.md]: Why did you move this tag?

Comment (iscai-msft) [reply specification/authorization/resource-manager/readme.java.md]: That was a mistake, moving it back now
